local-dev/configs/mtls.config

```diff
@@ -16,6 +16,7 @@ LHS_INTERNAL_ADVERTISED_HOST=localhost
 
 LHS_LISTENERS=MTLS:2023
 LHS_LISTENERS_PROTOCOL_MAP=MTLS:MTLS
+LHS_LISTENERS_AUTHENTICATION_MAP=MTLS:MTLS
```

Comment:
Can you add somewhere in a local-dev readme a tutorial about how to create a certificate that has a certain principal/subject name on it so that other people can test?